src/hevm/README.md

```diff
@@ -63,15 +63,47 @@ have GHC (the Haskell compiler) installed already, is to use
 [Stack](https://docs.haskellstack.org/en/stable/README/), which can
 take care of installing GHC for you.  These commands should work:
 
-    $ curl -sSL https://get.haskellstack.org/ | sh
-    $ git clone https://github.com/dapphub/hevm
-    $ cd hevm && stack setup && stack install
+```
```

Comment:
The rest of the README uses indentation to mark code blocks, so let's stick to that for now. We can change to code fences, if we want to, in a separate PR.